tests/framework/framework.go

```diff
@@ -75,41 +78,40 @@ func init() {
 
 // NewFramework makes a new framework and sets up the global BeforeEach/AfterEach's.
 // Test run-time flags are parsed and added to the Framework struct.
-func NewFramework(prefix string) *Framework {
+func NewFramework(prefix string) (*Framework, error) {
 	f := &Framework{
 		NsPrefix: prefix,
 	}
 
 	// handle run-time flags
 	if !flag.Parsed() {
 		flag.Parse()
-		// report flags values passed to test binary
-		fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
-		flag.Visit(func(f *flag.Flag) {
-			fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
-		})
-		fmt.Fprintf(GinkgoWriter, "**\n")
-		if kubectlPath != nil {
-			f.KubectlPath = *kubectlPath
-		}
-		if ocPath != nil {
-			f.OcPath = *ocPath
-		}
-		if cdiInstallNs != nil {
-			f.CdiInstallNs = *cdiInstallNs
-		}
-		if kubeConfig != nil {
-			f.KubeConfig = *kubeConfig
-		}
-		if master != nil {
-			f.Master = *master
-		}
+	}
+	// report flags values passed to test binary
+	fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
+	flag.Visit(func(f *flag.Flag) {
+		fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
+	})
+	fmt.Fprintf(GinkgoWriter, "**\n")
+	f.KubectlPath = *kubectlPath
+	f.OcPath = *ocPath
+	f.CdiInstallNs = *cdiInstallNs
+	f.KubeConfig = *kubeConfig
+	f.Master = *master
+	// It won't allow me to direct assign f.RestConfig, so using intermediate variable.
+	restConfig, err := f.LoadConfig()
```

Comment:
missing err check

Reply:
The check is below, but you are right, I should only assign if err == nil